src/actions/make/make_cpp.lua

```diff
@@ -98,6 +99,14 @@
 	end
 
 
+	function make.exePaths(cfg)
```

Comment:
unrelated, but I kind of had a hard time extracting it... it basically just adds support for "binpath", just like in MSVC.